website/pages/en/index.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react')
+
+const CompLibrary = require('../../core/CompLibrary.js')
+
+const MarkdownBlock = CompLibrary.MarkdownBlock /* Used to read markdown */
+
+const siteConfig = require(`${process.cwd()}/siteConfig.js`)
+
+function docUrl(doc, language) {
+  return `${siteConfig.baseUrl}docs/${language ? `${language}/` : ''}${doc}`
+}
+
+class Button extends React.Component {
+  render() {
+    return (
+      <div className="pluginWrapper buttonWrapper">
+        <a className="button" href={this.props.href} target={this.props.target}>
+          {this.props.children}
+        </a>
+      </div>
+    )
+  }
+}
+
+Button.defaultProps = {
+  target: '_self'
+}
+
+const SplashContainer = props => (
+  <div className="homeContainer">
+    <div className="homeSplashFade">
+      <div className="wrapper homeWrapper">{props.children}</div>
+    </div>
+  </div>
+)
+
+const Logo = props => (
+  <div className="projectLogo">
+    <img src={props.img_src} alt="Project Logo" />
+  </div>
+)
+
+const ProjectTitle = () => (
+  <h2 className="projectTitle">
+    {siteConfig.title}
+    <small>
+      <MarkdownBlock>
+        Official React bindings for [Redux](https://github.com/reduxjs/redux)
+      </MarkdownBlock>
+    </small>
+  </h2>
+)
+
+const PromoSection = props => (
+  <div className="section promoSection">
+    <div className="promoRow">
+      <div className="pluginRowBlock">{props.children}</div>
+    </div>
+  </div>
+)
+
+class HomeSplash extends React.Component {
+  render() {
+    const language = this.props.language || ''
+    return (
+      <SplashContainer>
+        <div className="inner">
+          <ProjectTitle />
+          <PromoSection>
+            <Button href={docUrl('introduction/quick-start', language)}>
+              Quick Start
+            </Button>
+            <Button href="https://github.com/reduxjs/redux-starter-kit">
+              Github
+            </Button>
+          </PromoSection>
+        </div>
+      </SplashContainer>
+    )
+  }
+}
+
+const Installation = () => (
+  <div
+    className="productShowcaseSection paddingBottom"
+    style={{ textAlign: 'center' }}
+  >
+    <h2>Installation</h2>
+    <MarkdownBlock>
+      Redux starter kit requires **Redux 4.0.0 or later.**
```

Comment:
This package already depends on Redux directly, so this line doesn't seem necessary.